backend/application/syson-application/src/test/java/org/eclipse/syson/JavaServiceIsCalledChecker.java

```diff
@@ -0,0 +1,201 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.syson;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.lang.reflect.Method;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.eclipse.acceleo.query.ast.Call;
+import org.eclipse.acceleo.query.parser.AstResult;
+import org.eclipse.acceleo.query.runtime.IQueryEnvironment;
+import org.eclipse.acceleo.query.runtime.IService;
+import org.eclipse.acceleo.query.runtime.Query;
+import org.eclipse.acceleo.query.runtime.QueryParsing;
+import org.eclipse.sirius.components.interpreter.SimpleCrossReferenceProvider;
+import org.eclipse.sirius.components.view.ChangeContext;
+import org.eclipse.sirius.components.view.Operation;
+import org.eclipse.sirius.components.view.RepresentationDescription;
+import org.eclipse.sirius.components.view.View;
+import org.eclipse.sirius.components.view.diagram.ConditionalNodeStyle;
+import org.eclipse.sirius.components.view.diagram.DiagramDescription;
+import org.eclipse.sirius.components.view.diagram.EdgeDescription;
+import org.eclipse.sirius.components.view.diagram.InsideLabelDescription;
+import org.eclipse.sirius.components.view.diagram.LabelEditTool;
+import org.eclipse.sirius.components.view.diagram.NodeDescription;
+import org.eclipse.sirius.components.view.diagram.OutsideLabelDescription;
+import org.eclipse.sirius.components.view.diagram.Tool;
+import org.eclipse.syson.sysml.helper.EMFUtils;
+
+/**
+ * Checks that a {@link Method} is called by at least one AQL expression in the given {@link View}.
+ *
+ * @author gdaniel
+ */
+public class JavaServiceIsCalledChecker {
+
+    private static final String AQL_PREFIX = "aql:";
```

Comment:
Please check that you have access to `org.eclipse.syson.util.AQLConstants` form here. If so, please use `AQLConstants.AQL` instead.

Reply:
done